.github/workflows/ceph-csi.yml

```diff
@@ -0,0 +1,51 @@
+name: ceph-csi
+
+on:
+  push:
+    paths:
+      - '.github/workflows/ceph-csi.yml'
+      - 'build/ceph-csi/**'
+      - '!build/ceph-csi/*.md'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: $(cat build/ceph-csi/.version)
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      # We need Docker, but not the Docker buildx bits?
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: latest
+
+      - name: Login to GitHub Docker Registry
+        run: echo "${DOCKERHUB_PASSWORD}" | docker login -u "${DOCKERHUB_USERNAME}" --password-stdin
+        env:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      # Repushing is necessary because a Docker manifest references images
+      # and those images have to exist in the registry beforehand.
```

Comment:
This has an annoying side effect of publishing single-arch images a la here: https://hub.docker.com/r/jamesorlakin/test-ceph. Docker registries have no 'delete tag' command to work around this as far as I can see.